Problem-1.java

```diff
@@ -0,0 +1,50 @@
+/*
+   
+Given an array of integers nums sorted in ascending order, find the starting and ending position of a given target value.*/
+
+//Time comlpexity - O(log n)
+//space complexity - O(1)
+// Did this code successfully run on Leetcode : Yes
+/*
+Any problem you faced while coding this : None
+
+*/
+
+// Your code here along with comments explaining your approach
+//two pointer approach where you search the target from the front and back
+class Solution {
+    //find the first occurance of the target
+    public int firstOccurance(int[] nums, int low, int high, int target)
```

Comment:
I would recommend you to have a flag and reuse the findFirstOccurence() instead of repeating the same logic in another method